index.html

```diff
@@ -4,18 +4,48 @@
     <meta charset="UTF-8" />
     <link rel="icon" href="public/icon-list.png" alt="list-icon" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="stylesheet" href="src/css/style.css" />
     <link
       href="https://fonts.googleapis.com/css2?family=Josefin+Sans:ital@1&display=swap"
       rel="stylesheet"
     />
-    <link rel="stylesheet" href="src/css/style.css" />
     <title>Abdullahi Todo List App</title>
   </head>
 
   <body class="bg-blue-900 text-lg md:mx-auto md:w-[600px]">
     <main role="main">
+      <section
+        class="fixed left-0 right-0 top-0 z-50 flex h-full items-center justify-center bg-blue-101 not-italic"
+        id="authentication-page"
+      >
+        <div class="relative flex items-center justify-center">
+          <div class="fixed bottom-[335px] w-[340px] rounded-lg bg-blue-0 p-[22px] shadow">
+            <p class="text-center text-2xl font-semibold">Login to your account</p>
+            <form id="form-button">
+              <label for="email">
+                <input
+                  class="my-2 w-[100%] rounded-lg border border-blue-300 p-3 shadow-sm outline-none ring-1 ring-offset-0 placeholder:text-blue-300 hover:bg-blue-0"
+                  type="email"
+                  id="email"
+                  title="Email Address"
+                  placeholder="Email Address"
+                  required
+                />
+              </label>
+              <button
+                class="flex w-full items-center justify-center rounded-lg bg-blue-101 p-3 font-bold text-blue-0 outline-none ring-1 hover:bg-blue-950 hover:text-blue-600 active:mx-[1px]"
+                type="submit"
+                id="sign-in-button"
+              >
+                Continue
+              </button>
+            </form>
+          </div>
+        </div>
+      </section>
+
       <section class="relative m-auto flex w-96 flex-col justify-center md:w-[34rem]">
```

Comment:
This should be display-none by default and it should be shown when a user is logged in

Reply:
Done